sql-statements/sql-statement-show-collation.md

```diff
@@ -53,33 +54,30 @@ SHOW COLLATION;
 | utf8mb4_general_ci | utf8mb4 |  45 |         | Yes      |       1 | PAD SPACE     |
 | utf8mb4_unicode_ci | utf8mb4 | 224 |         | Yes      |       8 | PAD SPACE     |
 +--------------------+---------+-----+---------+----------+---------+---------------+
-13 rows in set (0.00 sec)
+15 rows in set (0.000 sec)
 ```
 
-<CustomContent platform="tidb">
-
-When the new collation framework is disabled, only binary collations are listed.
+If the new framework for collations is disabled, only binary collations are listed.
 
 ```sql
 SHOW COLLATION;
 ```
 
 ```
-+-------------+---------+----+---------+----------+---------+---------------+
-| Collation   | Charset | Id | Default | Compiled | Sortlen | Pad_attribute |
-+-------------+---------+----+---------+----------+---------+---------------+
-| utf8mb4_bin | utf8mb4 | 46 | Yes     | Yes      |       1 | PAD SPACE     |
-| latin1_bin  | latin1  | 47 | Yes     | Yes      |       1 | PAD SPACE     |
-| binary      | binary  | 63 | Yes     | Yes      |       1 | NO PAD        |
-| ascii_bin   | ascii   | 65 | Yes     | Yes      |       1 | PAD SPACE     |
-| utf8_bin    | utf8    | 83 | Yes     | Yes      |       1 | PAD SPACE     |
-| gbk_bin     | gbk     | 87 | Yes     | Yes      |       1 | PAD SPACE     |
-+-------------+---------+----+---------+----------+---------+---------------+
-6 rows in set (0.00 sec)
++-------------+---------+-----+---------+----------+---------+---------------+
+| Collation   | Charset | Id  | Default | Compiled | Sortlen | Pad_attribute |
++-------------+---------+-----+---------+----------+---------+---------------+
+| utf8mb4_bin | utf8mb4 |  46 | Yes     | Yes      |       1 | PAD SPACE     |
+| latin1_bin  | latin1  |  47 | Yes     | Yes      |       1 | PAD SPACE     |
+| binary      | binary  |  63 | Yes     | Yes      |       1 | NO PAD        |
+| ascii_bin   | ascii   |  65 | Yes     | Yes      |       1 | PAD SPACE     |
+| utf8_bin    | utf8    |  83 | Yes     | Yes      |       1 | PAD SPACE     |
+| gbk_bin     | gbk     |  87 | Yes     | Yes      |       1 | PAD SPACE     |
+| gb18030_bin | gb18030 | 249 | Yes     | Yes      |       1 | PAD SPACE     |
++-------------+---------+-----+---------+----------+---------+---------------+
+7 rows in set (0.00 sec)
```

Comment:
Is it confirmed that we do not need to enclose this section using `<CustomContent platform="tidb">`?